# namespace of the pipeline's sensor message gateway
# string
namespace_gateway: "/cob_people_detection"
